src/fiatconnect/selectors.ts

```diff
@@ -20,3 +20,6 @@ export const cachedQuoteParamsSelector = (state: RootState) => state.fiatConnect
 export const schemaCountryOverridesSelector = (state: RootState) =>
   state.fiatConnect.schemaCountryOverrides
 export const personaInProgressSelector = (state: RootState) => state.fiatConnect.personaInProgress
+export const getCachedFiatConnectTransfer = (txHashToFind: string) => (state: RootState) => {
+  return state.fiatConnect.cachedTransfers.find(({ txHash }) => txHash === txHashToFind)
```

Comment:
are you sure both of these values will always have the same case?

Reply:
Not sure, will make lowercase just to be safe.